### PR TITLE
Ria 6490 non detained aaa

### PR DESCRIPTION
**Pull request by JamieHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-6490

### Change description ###
Changed the screen order for a case-officer/judge in the editafterappealsubmitted event to the following:

Decision letter details
Upload the Notice of Decision (If changed, then as is in case creation flow)
Basic details
Tell us about your client's nationality (If changed, then as is in case creation flow)
Your clients address (If changed, then as is in case creation flow)
        **New code states that this page will not appear if the appellant in in detention**
The appellant's contact preference (If changed, then as is in case creation flow)
        **New code states that this page will not appear if the appellant in in detention**
Litigation friend (If changed, then as is in case creation flow)
Other appeals (If changed, then as is in case creation flow)
Fee remissions (If changed, then as is in case creation flow)
How to pay (If client is not eligible for fee remission)-
Check your answer
Success page

Changes made are for a case that is age assessment and is not detained.
ageAssessment="Yes" AND appellantInDetention!="Yes"

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/EditAppealAfterSubmitHandler.java [hunk 1]
```diff
@@ -111,6 +112,10 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
                             parse(maybeDateEntryClearanceDecision
                                     .orElseThrow(() -> new RequiredFieldMissingException("dateEntryClearanceDecision is not present")));
                 }
+            } else if (appealType.equals(AppealType.AG)) {
+                Optional<String> mayBeDecisionLatter = asylumCase.read(DATE_ON_DECISION_LETTER, String.class);
```
Comment (Ryan-Nesbitt) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/EditAppealAfterSubmitHandler.java:116]: typo on 'maybeDecisionLetter' variable

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/NonDetainedAaaHandler.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                        .getCaseData();
+
+        YesOrNo appellantInUk = asylumCase
+                .read(APPELLANT_IN_UK, YesOrNo.class)
+                .orElseThrow(() -> new IllegalArgumentException("appellantInUk is missing"));
+
+        Optional<OutOfCountryDecisionType> outOfCountryDecisionType =
+                asylumCase.read(OUT_OF_COUNTRY_DECISION_TYPE, OutOfCountryDecisionType.class);
+
+        //Optional<YesOrNo> ageAssessment = asylumCase.read(AGE_ASSESSMENT, YesOrNo.class);
```
Comment (Ryan-Nesbitt) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/NonDetainedAaaHandler.java:52]: Are these commented out lines still needed or can they be removed?

**Review by Ryan-Nesbitt — CHANGES_REQUESTED** [review]
Left some comments

**Review by neha-aggarw — CHANGES_REQUESTED** [review]
Still needs a handler to wipe the details of LitigationFriend, if hasLitigationFriend is set to No. Thomas is working on it as part of RIA-6433. After that goes in, some changes will be required by this work as well.

**Pull request closed without merging**